lib/plugins/__init__.py

```diff
@@ -0,0 +1,61 @@
+from os.path import expanduser
+from lib import configuration
+import logging
+log = logging.getLogger('fabric.fabalicious.plugins')
+
+def init(root_folder, settings_name, folder_name, plugin_name, categories_filter):
```

Comment:
`init` is returning an array of plugins, How about we call it `loadPlugins` or `getPlugins`

Reply:
`root_folder` is being passed around so much, can we put it into `configuration` ?

Somewhere in `fabfile.py`, it is being set as `configuration.fabfile_basedir = root_folder`, however, when you try to read `configuration.fabfile_basedir`, it gives back the path where `fabfile.yaml` is present. Counter intuitive.